package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@keycloak/keycloak-admin-client",
-  "version": "16.0.0-dev.55",
+  "version": "16.0.0-dev.56",
```

Comment:
don't need to do this, this is done by the github action

Reply:
thanks @edewit!